Makefile

```diff
@@ -0,0 +1,73 @@
+LICENSE_CONFIG?="license-config.json"
+
+LICENSE:
+	@echo "you must have a LICENSE file" 1>&2
+	exit 1
+
+LICENSE_HEADER:
+	@echo "you must have a LICENSE_HEADER file" 1>&2
+	exit 1
+	
+	
+.PHONY: pretty
+pretty: node_modules/prettier
+	npm run format
+
+.PHONY: lint-fix
+lint-fix: node_modules/eslint
+	npm run lint:fix
+
+node_modules/eslint:
+	npm ci
+
+node_modules/mocha:
+	npm ci
```

Comment:
interesting, wouldnt this run `npm ci` multiple times?

Reply:
don't know, I just copied those targets out of the graphql project.  They're not normally run during the automated testing.